### PR TITLE
Fix edit employee

### PR DESCRIPTION
**Pull request by HemaSAli**
relates to #45 
in this pull request , we solve a problems in edit employee

### CODE REVIEW

File: src/database/query/deductionsQuery.js [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
-          + `from employees where id = ${employeeId}`,
-    )
-    .then((result) => {
-      const employeeDeductions = result[0][0];
-      resolve(employeeDeductions);
-    })
-    .catch(err => reject(err));
-});
+  const result = await sequelize.query(
+    'select deductions_health_insurance,deductions_loans,deductions_social_fund,(salary*(select saving_ratio from fixed_varibles where id = 1)) AS saving ,(select coalesce (SUM(payment),0) AS purchase_boxes from purchases_employees where employee_id = :employeeId )from employees where id = :employeeId', { replacements: { employeeId } },
```
Comment (ishak52) [inline src/database/query/deductionsQuery.js]: the line should contain maximum 100 char.

File: src/database/query/exemptions.js [hunk 2]
```diff
@@ -1,14 +1,9 @@
 const sequelize = require('../config/connection');
 
-const exemptions = (employee, variables) => new Promise((resolve, reject) => {
+const exemptions = async (employee) => {
   const { totalAllownace, employeeId } = employee;
-  const { dolar } = variables;
+  const result = await sequelize.query(`select exemption_university , exemption_house ,exemption_resident , ( select CASE when transportation > 0.1*(salary+${totalAllownace}) then (0.1*(salary+${totalAllownace})*(select saving_ratio from fixed_varibles where id = 1)) else (transportation * (select dolar from fixed_varibles where id = 1)) END AS "transportationsExemptions")from employees where id=${employeeId}`);
```
Comment (ishak52) [inline src/database/query/exemptions.js]: the line should contain maximum 100 char.

File: src/database/query/bonusQueryOneEmployee.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const sequelize = require('../config/connection');
+
+const bonusQueryOneEmployee = async (employeeId) => {
+  const id = employeeId;
+  const result = await sequelize.query('select id AS employee_id, name,id_number,salary ,CASE WHEN children>4 THEN 60 ELSE children*15 END AS child_allownace,'
++ 'CASE WHEN wife = true THEN 0 ELSE 30 END AS wife_allowance, transportation,allowance_gasoline,'
++ 'allowance_mobile,allowance_work,allowance_jerusalem,annual_rate, CASE WHEN children>4 THEN 60 ELSE children*15 END +'
++ '(CASE WHEN wife = true THEN 0 ELSE 30 END) + transportation+ allowance_gasoline+allowance_mobile+allowance_work+allowance_jerusalem +'
++ '(annual_rate*salary)  AS total_allownace from employees where id = :id', { replacements: { id } });
```
Comment (ishak52) [inline src/database/query/bonusQueryOneEmployee.js]: I need to know what this query does

File: src/views/employeeDetails.hbs [hunk 4]
```diff
@@ -237,6 +234,10 @@
             <div><input value="{{obj.exemptionUniversity}}" type="text" name="exemptionUniversity" class="input-field input-field-div4"
                 readonly></div>
           </li>
+          <li>
+            <span>اعفاء المواصلات</span>
+            <div><span>{{objExemption.transportationsExemptions}}</span></div>
```
Comment (ishak52) [inline src/views/employeeDetails.hbs]: Please explain what is `transportationsExemptions`?

**Comment by ishak52** [comment]
please fix conflicts and check requested changes @HemaSAli @othman-shamla @mohannadhanafi

File: src/database/query/bonusQueryOneEmployee.js (hunk 3 above)
Comment (HemaSAli) [reply src/database/query/bonusQueryOneEmployee.js]: I think I already explained it before !

File: src/views/employeeDetails.hbs (hunk 4 above)
Comment (othman-shamla) [reply src/views/employeeDetails.hbs]: translate it

**Pull request merged**